src/xpk/core/core.py

```diff
@@ -233,13 +233,16 @@ def parse_env_config(args, tensorboard_config, system: SystemCharacteristics):
 
   if not args.use_pathways:
     if args.debug_dump_gcs:
-      if 'XLA_FLAGS' in env:
+      if 'XLA_FLAGS' in env and '--xla_dump_to' in env['XLA_FLAGS']:
         raise ValueError(
-            'Conflict: XLA_FLAGS defined in both --debug_dump_gcs '
-            'and environment file. Please choose one way to define '
-            'XLA_FLAGS.'
+            'Conflict: --xla_dump_to defined in both --debug_dump_gcs '
+            'and environment file  via XLA_FLAGS. Please choose one way to define it.'
         )
-      env['XLA_FLAGS'] = '--xla_dump_to=/tmp/xla_dump/'
+      other_flags = ''
+      if 'XLA_FLAGS' in env:
+        other_flags = env['XLA_FLAGS']
+
+      env['XLA_FLAGS'] = '--xla_dump_to=/tmp/xla_dump/ ' + other_flags
```

Comment:
Not sure if /tmp should be hardcoded? Maybe os os.temp() or some function to get system tmp path?

Reply:
This path should be the same as in line 2122. I'm not sure if we can  pass around os.temp directory path between those two invocations.